src/main/java/ldbc/finbench/datagen/entities/edge/Apply.java

```diff
@@ -0,0 +1,67 @@
+package ldbc.finbench.datagen.entities.edge;
+
+import ldbc.finbench.datagen.entities.DynamicActivity;
+
+public class Apply implements DynamicActivity {
+    private long personId;
+    private long companyId;
+    private long loanId;
+    private long creationDate;
+    private long deletionDate;
+    private boolean isExplicitlyDeleted;
+
+    public Apply() {
```

Comment:
I see you define One Apply for both PersonApplyLoan and CompanyApplyLoan, I defined two apply relations for them. Maybe we should have simply communication and unify  definitions?

Reply:
Ok,I initially wanted to define these two types separately by the constructor,it is just a preliminary plan.

Reply:
You can merge the PR after you reach an agreement on the unified definitions of apply relations. My suggestion here is that you can define two separate relations for apply and then serialize them to the same "apply" label when output.